src/cryptoadvance/specter/templates/settings/tor_settings.jinja

```diff
@@ -15,37 +15,40 @@
 			<svg class="flex-shrink-0 inline w-6 h-6 mr-3" version="1.1" viewBox="0 0 64 64" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><!--Generated by IJSVG (https://github.com/iconjar/IJSVG)--><path d="M32.0022,55.9844l-1.04907e-06,-2.84217e-14c-13.2548,-5.79387e-07 -24,-10.7452 -24,-24c5.79387e-07,-13.2548 10.7452,-24 24,-24c13.2548,5.79387e-07 24,10.7452 24,24l2.13163e-14,-1.04907e-06c0,13.2548 -10.7452,24 -24,24Zm1.5,-32l3.82016e-08,-2.71086e-09c-1.79133,0.127116 -3.35165,-1.21031 -3.5,-3l6.50646e-08,-7.8524e-07c0.148296,-1.78972 1.70866,-3.12717 3.5,-3l-5.85322e-09,4.15529e-10c1.79134,-0.127171 3.3517,1.21028 3.5,3l-5.57104e-08,6.72365e-07c-0.148295,1.78976 -1.70872,3.12723 -3.5001,3Zm1.4206,6.7635l-2.7853,10.5093l-8.37179e-08,3.15667e-07c-0.270677,1.02062 0.262417,2.08281 1.2425,2.4757l-5.91015e-08,-2.36708e-08c0.379505,0.151996 0.628277,0.519688 0.6282,0.9285v0.323l8.52651e-14,4.05e-07c0,0.551933 -0.447167,0.999503 -0.9991,1h-2.9891l-4.04477e-08,1.96948e-10c-1.65938,0.00807986 -3.01111,-1.33056 -3.01919,-2.98993c-0.00127082,-0.260991 0.0314695,-0.521035 0.0973943,-0.773566l2.7852,-10.5094l2.79671e-08,-1.05468e-07c0.270652,-1.02067 -0.262537,-2.08289 -1.2427,-2.4757l-2.20542e-08,-8.83296e-09c-0.379505,-0.151996 -0.628276,-0.519688 -0.6282,-0.9285v-0.3229l-1.77636e-14,-9.40042e-08c-8.33514e-08,-0.552011 0.447289,-0.999614 0.9993,-1h2.9887l1.06497e-07,-5.28143e-10c1.65943,-0.00822943 3.01133,1.33033 3.01956,2.98976c0.00129458,0.261047 -0.0314356,0.52115 -0.0973639,0.773738Z" fill="currentColor" fill-rule="evenodd"></path></svg>
 			<span class="sr-only">Info</span>
 			<div>
-				{{ _("This anonymizes connections to block explorers, price providers and Electrum servers if Tor is activated here.") }}
+				{{ _("This anonymizes connections to block explorers and price providers (if you choose the Tor options for them) if Tor is activated here. The connection to Electrum servers is automatically via Tor if you have Tor enabled.") }}
```

Comment:
I'm surprised as i was not aware that this description is indeed correct with the current status-quo:
https://github.com/cryptoadvance/specter-desktop/blame/master/src/cryptoadvance/specter/util/price_providers.py#L128-L130

It doesn't make that much sense, though:
* Tor is a prerequisite to use spotbit-onion-services but you can also use to and access clearnet price-services
* There might be issues with bitstamp as it's not accepting requests from the tor-network. I thought that this is currently an issue (see #1554) but it shouldn't with the code i posted above.

I would suggest to leave it as is for now. So this comment was just for clarification.

Reply:
Copy changed in https://github.com/cryptoadvance/specter-desktop/pull/2304/commits/66a513683ac5f71a537ce68c2f876f08578315da